Airports/EGSH/Runway.txt

```diff
@@ -1 +1 @@
-09  27  089 269 N052.40.32.990 E001.16.08.590 N052.40.33.730 E001.17.46.580
\ No newline at end of file
+09  27  089 269 N052.40.32.870 E001.15.54.010 N052.40.33.730 E001.17.46.580
```

Comment:
```suggestion
09  27  089 269 N052.40.32.990 E001.16.08.600 N052.40.33.720 E001.17.46.580
```
You used the coordinates for I-NH for 09, not the runway. It was actually the 27 coordinates that changed

Reply:
oop may bad will correct it asap